security/pkg/pki/ca/controller/monitoring.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	configID = "SecretController"
+)
+
+var (
+	controllerLabels = []string{configID}
+
+	serviceAccountCreationCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "citadel",
+		Subsystem: "secret_controller",
+		Name:      "certs_created_due_to_service_account_creation",
```

Comment:
naming suggestion: `service_account_created_cert_count` or even `svc_acct_create_certs_count`

This will end up being `citadel_secret_controller_certs_created_due_to_service_acount_creation` currently. That's a lot.

Reply:
Done.